src/stringProcessing/relevantWords.js

```diff
@@ -9,6 +9,7 @@ let spanishFunctionWords = require( "../researches/spanish/functionWords.js" );
 let italianFunctionWords = require( "../researches/italian/functionWords.js" );
 let frenchFunctionWords = require( "../researches/french/functionWords.js" );
 let portugueseFunctionWords = require( "../researches/portuguese/functionWords.js" );
+let russianFunctionWords = require( "../researches/russian/functionWords.js" );
```

Comment:
Since more languages are being added, maybe it's nice to have one single import: `functionWords.js`. This file contains all the functionwords `requires`  for each language:

Example for this file (`researches/functionWords.js`):
```js
let germanFunctionWords = require( "../researches/german/functionWords.js" );
let englishFunctionWords = require( "../researches/english/functionWords.js" );
let dutchFunctionWords = require( "../researches/dutch/functionWords.js" );
let spanishFunctionWords = require( "../researches/spanish/functionWords.js" );
let italianFunctionWords = require( "../researches/italian/functionWords.js" );
let frenchFunctionWords = require( "../researches/french/functionWords.js" );
let portugueseFunctionWords = require( "../researches/portuguese/functionWords.js" );
let russianFunctionWords = require( "../researches/russian/functionWords.js" );

module.exports = function() {
    return {
       de: germanFunctionWords,
       nl: dutchFunctionWords,
       fr: frenchFunctionWords,
       es: spanishFunctionWords,
       it: italianFunctionWords,
       pt: portugueseFunctionWords,
       ru: russianFunctionWords,
    }
};
```

By having that file you only have to import one single file:
```js
let functionWordLists = require( "../researches/functionWords.js" );
```

The big `switch` statement in the `getRelevantWords` function below can be replaced by a simple if:
```js
let language = getLanguage( locale );
if ( ! functionWordLists.hasOwnProperty( language ) {
    language = "en";
}   

let functionWords = functionWordLists[ language ];
```

The whole advantage of this approach is easy extensibility. If a new language like Swedish is being added we only have to change the file that contains all language specific lists.

Maybe it's not something for now. But I think we should consider doing something like this.

Reply:
Edited the scripts following your suggestion.